_hardware/04-CompatibilityCharts/03-Storage/03-NVMe.md

```diff
@@ -26,39 +26,43 @@ has_toc: false
 </p>
 <br>
 
-{: .internalnote }
-This sections is currently a work in progress, and information here may be incorrect. The intended layout is: Recommended NVMEs known to work extremely well, Various Brands and associated known working NVMEs, finally the Unsupported NVMEs section, which is used to keep track with community support.
+{: .important }
+NVMe storage devices can sometimes require additional patches or kexts for full functionality, particularly when it comes to ensuring proper support for TRIM and enabling the drive to be recognized correctly by macOS.
 # Recommended NVMes
 
 | Drive Name | Notes |
-| placeholder | placeholder |
-| placeholder | placeholder |
-| placeholder | placeholder |
-| placeholder | placeholder |
+| WD_BLACK SN750 | One of the best choices. |
+| Western Digital SN700 | One of the best choices. |
+| WD_BLACK SN850/SN850X| One of the best choices. |
 
 # Samsung
 
+{: .important }
+Many Samsung NVMe drives are known to cause issues, so it's best to avoid using them if possible.
+
 | Drive Name | Notes |
-| placeholder | placeholder |
-| placeholder | placeholder |
-| placeholder | placeholder |
-| placeholder | placeholder |
+| Samsung 950 Pro | Should be used as an storage-only NVMe |
+| Samsung 960 Evo/Pro | Should be used as an storage-only NVMe |
+| Samsung 970 Evo/Pro | Should be used as an storage-only NVMe |
+| Samsung 980 Pro | Should be used as an storage-only NVMe |
```

Comment:
that leaves too many questions unanswered

Reply:
i believe the SSD Hall of Fame can be used as a reference point